### ISSUE TITLE
AttributeError: 'ParsedRequirement' object has no attribute 'req'

### ISSUE THREAD

**Issue by sirex** [socialwifi/jsonapi-requests#52]
```
> pip download --no-deps --no-binary=:all: jsonapi-requests==0.6.1
Collecting jsonapi-requests==0.6.1
  Using cached jsonapi-requests-0.6.1.tar.gz (12 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-download-cg0e255r/jsonapi-requests/s
etup.py'"'"'; __file__='"'"'/tmp/pip-download-cg0e255r/jsonapi-requests/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__fi
le__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-ba
se /tmp/pip-pip-egg-info-y26dvko1
         cwd: /tmp/pip-download-cg0e255r/jsonapi-requests/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-download-cg0e255r/jsonapi-requests/setup.py", line 24, in <module>
        install_requires=[str(ir.req) for ir in parse_requirements('base_requirements.txt', session=False)],
      File "/tmp/pip-download-cg0e255r/jsonapi-requests/setup.py", line 24, in <listcomp>
        install_requires=[str(ir.req) for ir in parse_requirements('base_requirements.txt', session=False)],
    AttributeError: 'ParsedRequirement' object has no attribute 'req'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Related issue: https://github.com/pypa/pip/issues/8188

**Comment by jgoclawski** [comment]
@sirex Thanks for letting us know, we need to change the code depending on internal pip API, which apparently was changed.
Out of curiosity: why would you explicitly disable installation through a wheel? Different architecture?

**Comment by jgoclawski** [comment]
Fixed in 0.6.2.
```
$ pip install --no-deps --no-binary=:all: jsonapi-requests==0.6.2
Collecting jsonapi-requests==0.6.2
  Downloading jsonapi-requests-0.6.2.tar.gz (12 kB)
Skipping wheel build for jsonapi-requests, due to binaries being disabled for it.
Installing collected packages: jsonapi-requests
    Running setup.py install for jsonapi-requests ... done
Successfully installed jsonapi-requests-0.6.2
```

**Comment by sirex** [comment]
Our CI downloads source packages, builds wheels and uploads those wheels into internal pypi. That is why I need source packages.
